### ISSUE TITLE
Register a strategy for `libcst.MatchSingleton`

### ISSUE THREAD

**Issue by 0-wiz-0** [Zac-HD/hypothesmith#21]
I've packaged hypothesmith 0.2.0 and ran the self tests, and I got:
```
_____________________________________________________________________ test_source_code_from_libcst_node_type[MatchSingleton] _____________________________________________________________________
tests/test_cst.py:23: in test_source_code_from_libcst_node_type
    @given(data=st.data())
E   hypothesis.errors.Unsatisfiable: Unable to satisfy assumptions of test_source_code_from_libcst_node_type
------------------------------------------------------------------------------------------- Hypothesis -------------------------------------------------------------------------------------------
You can add @seed(77441592302281572436730079892928709310) to this test or run pytest with --hypothesis-seed=77441592302281572436730079892928709310 to reproduce this failure.
```
The installed dependencies are libcst-0.4.1, lark-parser-0.12.0, and hypothesis 6.36.1, with python 3.10.2; on NetBSD/amd64 in case it matters.

**Comment by Zac-HD** [comment]
Thanks for reporting this - Hypothesmith doesn't have more-than-accidental support for the new grammar features in Python 3.10, and it looks like we'll need it in this case.

No timeline on that for a hobby project, but it would be entirely appropriate to skip this test until it's fixed.

**Comment by Zac-HD** [comment]
Fixed by #24 🐌
